### PR TITLE
Updating release instruction with ReleaseRapidProject

### PR DESCRIPTION
**Pull request by suztomo**
With cl/351500804, this PR updates the release instruction with ReleaseRapidProject for both "bom" and "dependencies".

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -116,10 +116,27 @@ clientdir="$(p4 g4d -- "${citcclient?}")"
...
+    --project_name=cloud-java-tools-cloud-opensource-java-dependencies-kokoro-release \
+    --version=${VERSION} --committish_suffix=${SUFFIX} &
+  blaze-bin/java/com/google/cloud/java/tools/ReleaseRapidProject \
+    --project_name=cloud-java-tools-cloud-opensource-java-enforcer-rules-release \
+    --version=${VERSION} --committish_suffix=${SUFFIX} &
+  blaze-bin/java/com/google/cloud/java/tools/ReleaseRapidProject \
+    --project_name=cloud-java-tools-cloud-opensource-java-gradle-plugin-kokoro-release \
+    --version=${VERSION} --committish_suffix=${SUFFIX} &
+  wait
+fi
```
Comment (netdpb) [inline scripts/release.sh:139]:
```suggestion
RELEASE_RAPID_PROJECT=java/com/google/cloud/java/tools:ReleaseRapidProject
blaze build "${RELEASE_RAPID_PROJECT}"

release_rapid_project() {
  local project="$1"
  "blaze-bin/${RELEASE_RAPID_PROJECT/://}" \
      --project_name="cloud-java-tools-cloud-opensource-java-${project}-release" \
      --version="${VERSION}" --committish_suffix="${SUFFIX}"
}

if [[ "${SUFFIX}" = bom ]]; then
  release_rapid_project bom-kokoro
else
  # Run the Rapid projects concurrently
  release_rapid_project parent-kokoro &
  release_rapid_project dependencies-kokoro &
  release_rapid_project enforcer-rules &
  release_rapid_project gradle-plugin-kokoro &
  wait
fi
```

Comment (suztomo) [reply scripts/release.sh]: Thank you. Updated.

File: scripts/release.sh [hunk 2]
```diff
@@ -116,25 +116,23 @@ clientdir="$(p4 g4d -- "${citcclient?}")"
...
-      --version=${VERSION} --committish_suffix=${SUFFIX}
+RELEASE_RAPID_PROJECT=java/com/google/cloud/java/tools:ReleaseRapidProject
+blaze build "${RELEASE_RAPID_PROJECT}"
+
+release_rapid_project() {
+  local project="$1"
+  "blaze-bin/${RELEASE_RAPID_PROJECT/://}" \
+      --project_name="cloud-java-tools-cloud-opensource-java-${project}-release" \
+      --version="${VERSION}" --committish_suffix="${SUFFIX}"
+}
```
Comment (netdpb) [inline scripts/release.sh:127]: Probably should have a blank line after this function declaration.

**Review by netdpb — APPROVED** [review]

File: scripts/release.sh (hunk 2 above)
Comment (suztomo) [reply scripts/release.sh:127]: Added a blank line.

File: RELEASING.md [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
+
+    * Run `gh auth login` to register your desktop with github.
+
+* Clone this repository onto your corp desktop, Ubiquity instance, or CloudTop. Do not use a laptop or personal machine as the release requires google3 access.
+
+## Release
+
 To release artifacts 'dependencies-parent', 'dependencies', 'linkage-checker-enforcer-rules',
-and 'gradle-plugin', run `prepare_release.sh` with `dependencies` argument:
+and 'gradle-plugin', run `script/release.sh` with `dependencies` argument in corp desktop:
```
Comment (elharo) [inline RELEASING.md:17]: delete "in corp desktop"

File: RELEASING.md [hunk 4]
```diff
@@ -1,22 +1,32 @@
...
 ```
 
 Note: Google Cloud Libraries BOM release procedure [boms/cloud-oss-bom/RELEASING.md](
 boms/cloud-oss-bom/RELEASING.md) is not part of this document.
 
-Create a PR for the release, and get it approved.
-
-Continue to Rapid workflow: [Cloud Tools for Java Development Practices: Releasing](
-https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md#run-the-rapid-workflow)
+This script creates a PR for the release and initiates Rapid project workflows listed below.
```
Comment (elharo) [inline RELEASING.md:29]: the Rapid workflows

File: boms/cloud-oss-bom/RELEASING.md [hunk 5]
```diff
@@ -77,7 +77,9 @@ run this command from a g4 client to retry the Rapid build without going all the
 back to the beginning:
 
 ```
-$ blaze run java/com/google/cloud/java/tools:ReleaseBom -- --version=${VERSION}
+$ blaze run java/com/google/cloud/java/tools:ReleaseRapidProject -- \
+    --project_name=cloud-java-tools-cloud-opensource-java-bom-kokoro-release \
+    --version=${VERSION} --committish_suffix=bom
```
Comment (elharo) [inline boms/cloud-oss-bom/RELEASING.md:82]: "committish_suffix" feels wrong as if it's expsing implementation details in the UI. This should probably be either suffix or more likely the whole commitish.

File: RELEASING.md (hunk 3 above)
Comment (suztomo) [reply RELEASING.md]: Done

File: RELEASING.md (hunk 4 above)
Comment (suztomo) [reply RELEASING.md]: Done

File: boms/cloud-oss-bom/RELEASING.md (hunk 5 above)
Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md]: Updated for committish. Removed version flag.

**Review by suztomo — COMMENTED** [review]
@elharo PTAL. Updated cl/351500804 too.

**Review by elharo — APPROVED** [review]

**Pull request merged**